packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -26,10 +28,16 @@ export interface ISummaryStats {
 
 /**
  * Represents the summary tree for a node along with the statistics for that tree.
+ *
+ * @remarks
  * For example, for a given data store, it contains the data for data store along with a subtree for
  * each of its DDS.
- * Any component that implements IChannelContext, IFluidDataStoreChannel or extends SharedObject
+ *
+ * Any component that implements {@link @fluidframework/datastore#IChannelContext}, {@link IFluidDataStoreChannel}
+ * or extends {@link @fluidframework/shared-object-base#SharedObject}
```

Comment:
Again concerns about links to things in packages that this one doesn't depend on.

Reply:
Yeah, probably better to just remove this part of the comment altogether then.